apps/emqx/src/emqx_vm.erl

```diff
@@ -62,6 +62,7 @@
 
 -define(PROCESS_INFO_KEYS, [initial_call,
                             current_function,
```

Comment:
actually, with `current_stacktrace`, there is no need for `current_function` anymore.

Reply:
Ok! I'll remove `current_function` then.